Users/Tutorial.rst

```diff
@@ -28,7 +28,7 @@ use with this sample code and may need minor adjustments.
 Let's Start!
 ------------
 
-There are two options how to let coala know what kind of analysis it
+There are two options used to let coala know what kind of analysis it
```

Comment:
ah I think what he means is that you can use the CLI and config files... I feel like we should completely rework this tutorial it seems to be in a bad state :/